cmd/operator/main.go

```diff
@@ -79,6 +79,7 @@ func (t *tags) Type() string {
 func Main() int {
 	flagset := flag.CommandLine
 	namespace := flagset.String("namespace", "openshift-monitoring", "Namespace to deploy and manage cluster monitoring stack in.")
+	namespaceSelector := flagset.String("namespace-selector", "openshift.io/cluster-monitoring=true", "Selector for namespaces to monitor.")
```

Comment:
very very small nit: not sure if this needs to be configurable

Reply:
As a side note, within openshift users will never be able to configure this, this is only configurable so we have the ability to change it should we decide on a different labeling/selection scheme.

Reply:
If I happen to run the monitoring operator on any other K8s cluster which is probably not OpenShift, it could be that I want to change this setting.

Reply:
As of today, this operator will not work on anything but openshift. This may change in the future, but may take a long time to be a reality, or maybe never.